docs/reference/release-notes/highlights-6.7.0.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[release-highlights-6.7.0]]
+== 6.7.0 release highlights
+++++
+<titleabbrev>6.7.0</titleabbrev>
+++++
+
+See also <<release-notes-6.7.0,{es} 6.7.0 release notes>>.
+
+[float]
+=== Cross-cluster replication
+
+We have introduced cross-cluster replication in 6.5.0 as a beta feature and it
+is now mature enough that we make it generally available. Cross-cluster
+replication enables you to replicate indices that exist in remote clusters to
+your local cluster. You create an index in your local cluster (known as
+a _follower index_) that follows an index (known as a _leader index_) in a
+remote cluster. You can also automatically follow indices in a remote cluster
+that match a pattern. The individual write operations that occur on the leader
+indices are then replayed on the follower indices. This functionality is useful
+for replicating data to a second cluster for disaster recovery purposes and for
+geo-proximity so that reads can be served locally.
+
+For more information, see {stack-ov}/xpack-ccr.html[Cross-cluster replication]
+and <<ccr-apis>>.
+
+=== Index lifecycle management
+
+We have introduced index lifecycle management in 6.6.0 as a beta feature and it
+is now mature enough that we make it generally available. The index lifecycle
```

Comment:
Maybe the same wording from CCR: "We introduced index lifecycle management in 6.6.0 as a beta feature, it
is now mature enough to declare the feature GA (general availability).